include/language.php

```diff
@@ -628,7 +546,17 @@ public function get_display_flag() {
 	 * @return string
 	 */
 	public function get_display_flag_url() {
-		return empty( $this->custom_flag_url ) ? $this->flag_url : $this->custom_flag_url;
+		$flag_url = empty( $this->custom_flag_url ) ? $this->flag_url : $this->custom_flag_url;
+
+		/**
+		 * Filters flag URL. The custom one is passed if exists.
+		 *
+		 * @since 3.4
+		 *
+		 * @param string       $flag_url Flag URL.
+		 * @param PLL_Language $language Current language object.
+		 */
+		return apply_filters( 'pll_get_display_flag_url', $flag_url, $this );
```

Comment:
Why do we need the language object in this filter ?

Reply:
Today `pll_get_display_flag_url` is only filtered by `site_url()`. But I added `$this` for our future self and for our users. Do you think it'd be better without it?